app/Auth/CustomGate.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Rogue\Auth;
+
+use Rogue\Models\User;
+use Illuminate\Support\Arr;
+use Illuminate\Auth\Access\Gate;
+use Illuminate\Contracts\Auth\Access\Gate as GateContract;
+
+class CustomGate extends Gate implements GateContract
+{
+    protected function raw($ability, $arguments = [])
+    {
+        if (! $user = $this->resolveUser()) {
+            // return false;
+            $user = new User;
```

Comment:
I think we need to return a `$user` here so the rest of the code works, but we probably want to set a `$user` from `Illuminate\Contracts\Auth\Authenticatable;` and not `Rogue\Models\User` - is this correct? I don't know if this is something we can do though... 

When I try to do set a `$user` from `Illuminate\Contracts\Auth\Authenticatable`, I get this error: 
```
"Cannot instantiate interface Illuminate\Contracts\Auth\Authenticatable"
``` 
and when I try to set a `$user` from `Rogue\Models\User`, I get: 
```
"Undefined variable: errors (View: /home/vagrant/DoSomething.org/rogue/resources/views/layouts/master.blade.php) (View: /home/vagrant/DoSomething.org/rogue/resou ▶"
```

Reply:
@DFurnes it doesn't like that either :-/ Got this error: 

```
"Type error: Argument 1 passed to Rogue\Policies\PostPolicy::show() must be an instance of Illuminate\Contracts\Auth\Authenticatable, null given, called in /home ▶"
```